templates/taxi/car_confirm_delete.html

```diff
@@ -7,4 +7,4 @@ <h1>Delete car?</h1>
 
     <input type="submit" value="Yes" class="btn btn-danger">
```

Comment:
According to the checklist, there should be a 'Cancel' button in the form apart from the 'Delete' one. The 'Cancel' button will lead to the previous page the user was on. Please add a 'Cancel' button here.